### DISCUSSION TITLE
Quarkus with Azure App Insights

### DISCUSSION THREAD

**Discussion by SebDeLille** [quarkusio/quarkus#27695] [Community]
Hello,

Has anybody already use Azure app insights connector with a quarkus app? The examples I see on docs seem not work. I 've done some try with applicationinsights-agent-3.3.1 and applicationinsights-agent-3.2.11 but without results

Thanks in advance,
Sebastien

**Comment by yroffin** [comment]
Did you receive any support on your request, i've the same question :)

**Comment by SebDeLille** [comment]
Hello,

Problem solved.

To use app insights agent, it must be declared as dependency in the pom.xml. 
and the javaagent must be declared in configuration part of quarkus-maven-plugin

**Reply by gsmet** [reply]
Could you paste an example for the others struggling with this? Thanks.

**Reply by SebDeLille** [reply]
I use this project to solve my issue : https://github.com/mtaghavi2005/quarkus-application-insight
